### PR TITLE
adding Windows config file sample

### LINKED ISSUE #22709: daemon.json documentation sample is not valid on Windows
The list of allowed configuration options in [this document](https://docs.docker.com/engine/reference/commandline/daemon/#daemon-configuration-file) is not applicable to Windows. Several of the options do not work on Windows. I think this requires minor clarification, perhaps a sample that is applicable to Windows:

When used as is, the following is returned:

```none
PS C:\> dockerd
time="2016-05-12T15:40:59-07:00" level=fatal msg="unable to configure the Docker daemon with file C:\\ProgramData\\docker\\config\\daemon.json: the following directives don't match any configuration option: iptables, bip, icc, api-cors-headers, exec-root, default-gateway-v6, ipv6, cgroup-parent, ip-mask, ip, userland-proxy, default-gateway, ip-forward, selinux-enabled, userns-remap, fixed-cidr-v6\n"
```

Here is a working sample:

```json
{
    "authorization-plugins": [],
    "dns": [],
    "dns-opts": [],
    "dns-search": [],
    "exec-opts": [],
    "storage-driver": "",
    "storage-opts": [],
    "labels": [],
    "log-driver": "", 
    "mtu": 0,
    "pidfile": "",
    "graph": "",
    "cluster-store": "",
    "cluster-advertise": "",
    "debug": true,
    "hosts": ["tcp://0.0.0.0:2376", "npipe://"],
    "log-level": "",
    "tlsverify": true,
    "tlscacert": "C:\\ProgramData\\docker\\certs.d\\ca.pem",
    "tlscert": "C:\\ProgramData\\docker\\certs.d\\server-cert.pem",
    "tlskey": "C:\\ProgramData\\docker\\certs.d\\server-key.pem",
    "group": "",
    "default-ulimits": {},
    "bridge": "",
    "fixed-cidr": "",
    "raw-logs": false,
    "registry-mirrors": [],
    "insecure-registries": [],
    "disable-legacy-registry": false
}

```
If this makes sense, and is in line with the docs scope / purpose, I am happy write up and propose the changes.

neilp

### PR DESCRIPTION
**Pull request by neilpeterson**
**\- What I did**

Documentation update to add Windows example of daemon.json. This change fixes [#22709](https://github.com/docker/docker/issues/22709).

**\- How I did it**

Updated document. Added H3 for Linux and Windows so that each section is present in right hand navigation.

**\- How to verify it**

Verify document rendering.

**\- Description for the changelog**

Added documentation that includes an example daemon.json file for Windows.

Signed-off-by: Neil Peterson neilpeterson@outlook.com

### CODE REVIEW

File: docs/reference/commandline/dockerd.md [hunk 1]
```diff
@@ -882,19 +882,20 @@ the `--cgroup-parent` option on the daemon.
...
 silently ignore changes introduced in configuration reloads.
 For example, the daemon fails to start if you set daemon labels
-in the configuration file and also set daemon labels via the `--label` flag.
+in the configuration file and also set daemon labels via the `--label` flag. Options that are not present in the file are ignored when the daemon starts.
 
-Options that are not present in the file are ignored when the daemon starts.
-This is a full example of the allowed configuration options in the file:
+### Linux configuration file
+
+The default location of the configuration file on Linux is `/etc/docker/daemon.json`.
```
Comment (thaJeztah) [inline docs/reference/commandline/dockerd.md]: Perhaps we should mention that a different path can be specified through the `--config-file` option?

File: docs/reference/commandline/dockerd.md [hunk 2]
```diff
@@ -951,6 +952,46 @@ This is a full example of the allowed configuration options in the file:
 }
 ```
 
+### Windows configuration file
+
+The default location of the configuration file on Linux is `%programdata%\docker\config\daemon.json`.
```
Comment (thaJeztah) [inline docs/reference/commandline/dockerd.md]: Same here

File: docs/reference/commandline/dockerd.md [hunk 3]
```diff
@@ -882,19 +882,20 @@ the `--cgroup-parent` option on the daemon.
...
-on Linux and `%programdata%\docker\config\daemon.json` on Windows.
+of the flag name, e.g., `labels` for the `label` flag.
 
 The options set in the configuration file must not conflict with options set
 via flags. The docker daemon fails to start if an option is duplicated between
 the file and the flags, regardless their value. We do this to avoid
 silently ignore changes introduced in configuration reloads.
 For example, the daemon fails to start if you set daemon labels
-in the configuration file and also set daemon labels via the `--label` flag.
+in the configuration file and also set daemon labels via the `--label` flag. Options that are not present in the file are ignored when the daemon starts.
```
Comment (thaJeztah) [inline docs/reference/commandline/dockerd.md]: Can you wrap this to 80-chars?

File: docs/reference/commandline/dockerd.md (hunk 1 above)
Comment (thaJeztah) [inline docs/reference/commandline/dockerd.md]: Also, can you wrap this to 80-chars?

**Comment by thaJeztah** [comment]
Thanks @neilpeterson, I some suggestions, but otherwise I think this is good to go

**Comment by neilpeterson** [comment]
@thaJeztah, thanks, suggestions have been incorporated.

File: docs/reference/commandline/dockerd.md [hunk 4]
```diff
@@ -882,19 +882,23 @@ the `--cgroup-parent` option on the daemon.
...
-in the configuration file and also set daemon labels via the `--label` flag.
-
+in the configuration file and also set daemon labels via the `--label` flag. 
 Options that are not present in the file are ignored when the daemon starts.
-This is a full example of the allowed configuration options in the file:
+
+### Linux configuration file
+
+The default location of the configuration file on Linux is 
+`/etc/docker/daemon.json`. The `--conflig-file` flag can be used to specify a
```
Comment (StefanScherer) [inline docs/reference/commandline/dockerd.md]: Just spotted a small typo. `--conflig-file` -> `--config-file` and some lines below.

**Comment by thaJeztah** [comment]
Thanks @neilpeterson can you;

- fix the typos that @StefanScherer spotted
- wrap the new content to 80-chars
- squash your commits (I think this change doesn't need multiple commits, just a single one should be ok)

**Comment by thaJeztah** [comment]
oops, browser cache, you already wrapped :blush:

**Comment by neilpeterson** [comment]
I've made a mess on the PR / Squash. Can't spend any more time on it. Will close and re-open.

**Pull request closed without merging**